my-sms/app/views/students/_form.html.erb

```diff
@@ -1,45 +1,76 @@
-<%= form_for(@student) do |f| %>
-  <% if @student.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@student.errors.count, "error") %> prohibited this student from being saved:</h2>
+<%= form_for @student, :html => { :class => "form-horizontal student" } do |f| %>
 
-      <ul>
-      <% @student.errors.full_messages.each do |msg| %>
-        <li><%= msg %></li>
-      <% end %>
-      </ul>
+  <% if @student.errors.any? %>
+    <div id="error_expl" class="panel panel-danger">
+      <div class="panel-heading">
+        <h3 class="panel-title"><%= pluralize(@student.errors.count, "error") %> prohibited this student from being saved:</h3>
+      </div>
+      <div class="panel-body">
+        <ul>
+        <% @student.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+        </ul>
+      </div>
     </div>
```

Comment:
Another way to do this is to put it in a flash and define a way to display the flash on "standard" views/pages, making this error view a bit more re-usable

Reply:
Did this in 4f9c2f9e9c0210c5872efd072f6431d5efcf61fd
Defined a way to display flashes on standard views/pages and automatically created error flashes if there were errors on the Student model.